common/test/resources/timeout-adapter.js

```diff
@@ -0,0 +1,43 @@
+// Preprocessing of the Karma configuration's client.args parameter.
+
+com = com || {};
+com.keyman = com.keyman || {};
+com.keyman.karma = com.keyman.karma || {};
+
+(function() {
+  const testconfig = window['testconfig'] = {};
```

Comment:
This variable had different names depending on the project using it:

- Web's unit tests:  `kmwconfig`
- LMLayer unit tests:  `config` (not to be confused with the Karma config)

At least one of those spots would need to change anyway.

#6970 will be updated with this PR's changes, and it'll want to reference this object as well.  I'd gone with `testconfig` there, since `kmwconfig` doesn't really fit.